### PR TITLE
Progress so far

### PR DESCRIPTION
**Pull request by danderson0851**
Please review my code progress so far and send me corrections / advice

### CODE REVIEW

File: LoggingKata.Test/ParserTest.cs [hunk 1]
```diff
@@ -10,10 +10,56 @@ namespace LoggingKata.Test
...
+            //Assert
+            Assert.IsNull(failWithInvalidInput);
+            Assert.IsNull(failWithOtherInvalidInput);
+
+        }
         [Test]
         public void ShouldParseLine()
         {
-            //TODO: Complete ShouldParseLine
+            //Arrange
```
Comment (codytwinton) [inline LoggingKata.Test/ParserTest.cs]: Valid lat and long with no description should still parse

File: LoggingKata/Program.cs [hunk 2]
```diff
@@ -16,21 +17,50 @@ class Program
...
         static void Main(string[] args)
         {
-            if (args.Length == 0)
+            var myFile = Environment.CurrentDirectory + "\\Taco_Bell-US-AL-Alabama.csv";
+            var lines = File.ReadAllLines(myFile);
+
+            if (lines.Length == 0)
+            {
+                Console.WriteLine("You must provide a filename as an argument");
+                Logger.Fatal("Cannot import without filename specified as an argument");
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: You logging for length 1 and 0 are the same.

File: LoggingKata/Program.cs [hunk 3]
```diff
@@ -16,21 +17,50 @@ class Program
...
             Logger.Info("Log initialized");
-            var lines = File.ReadAllLines(args[0]);
+            
             var parser = new TacoParser();
             var locations = lines.Select(line => parser.Parse(line));
 
             //TODO:  Find the two TacoBells in Alabama that are the furthurest from one another.
             //HINT:  You'll need two nested forloops
 
+            /*
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: Missing implementation

File: LoggingKata.Test/ParserTest.cs (hunk 1 above)
Comment (danderson0851) [reply LoggingKata.Test/ParserTest.cs]: Done

File: LoggingKata/Program.cs (hunk 2 above)
Comment (danderson0851) [reply LoggingKata/Program.cs]: Done

**Comment by danderson0851** [comment]
I think i addressed all the changes requested, please give me your feedback when you can

File: LoggingKata.Test/ParserTest.cs [hunk 4]
```diff
@@ -4,16 +4,68 @@
...
 namespace LoggingKata.Test
 {
     [TestFixture]
     public class TacoParserTestFixture
     {
+        [Test]
+        public void TwoOfThreeShouldParse()
+        {
+            //Arrange
+            var TwoOfThreeTest = new TacoParser();
```
Comment (codytwinton) [inline LoggingKata.Test/ParserTest.cs]: This parser variable should be camelCase, not PascalCase.

File: LoggingKata.Test/ParserTest.cs [hunk 5]
```diff
@@ -4,16 +4,68 @@
...
+            //Act
+            var valueReturned = TwoOfThreeTest.Parse("-82.225,31.004");
+            //Assert
+            Assert.IsNull(valueReturned);
```
Comment (codytwinton) [inline LoggingKata.Test/ParserTest.cs]: Should be `Assert.IsNotNull`

File: LoggingKata/TacoBell.cs [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LoggingKata
+{
+    class TacoBell : ITrackable
+    {
+        public string Name { get; set; }
+        public Point Location { get; set; }
+
```
Comment (codytwinton) [inline LoggingKata/TacoBell.cs]: No whitespace needed here.

File: LoggingKata/TacoParser.cs [hunk 7]
```diff
@@ -1,26 +1,43 @@
...
-        public TacoParser()
+        private static readonly ILog Logger = LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+
+        public ITrackable Parse(string line)
         {
+            if (string.IsNullOrEmpty(line)) { return null; }
+            var cells = line.Split(',');
 
-        }
+            if (cells.Length < 3)
```
Comment (codytwinton) [inline LoggingKata/TacoParser.cs]: Should be 2, not 3

File: LoggingKata/TacoParser.cs [hunk 8]
```diff
@@ -1,26 +1,43 @@
...
+            {
+                var l1 = Double.Parse(cells[0]);
+                var l2 = Double.Parse(cells[1]);
+                var tbName = cells[2];
 
-        public ITrackable Parse(string line)
-        {
-            //DO not fail if one record parsing fails, return null
-            return null; //TODO Implement
+                var tacoBell = new TacoBell();
```
Comment (codytwinton) [inline LoggingKata/TacoParser.cs]: Probably better to use the object initializer.

**Comment by danderson0851** [comment]
Got it, thanks

**Pull request closed without merging**